lib/status/status.go

```diff
@@ -105,22 +105,6 @@ func FromCluster(ctx context.Context, operator ops.Operator, cluster ops.Site, o
 	return status, nil
 }
 
-// Check classifies this cluster status as error
-func (s Status) Check() error {
```

Comment:
Not sure why this was added. This breaks robotest's ability to quickly determine if an operation has failed.
Although `robotest` can use the full `gravity status` to do the same, the behavior we had with `--operation-id= --quiet` was to output just the operation status which does not work if the cluster is degraded and `Cluster status: degraded` is output instead.